cards/condyle-fracture-lateral/condyle-fracture-lateral

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
This whole file should be named `card.md`. The folder name is great. Within the folder the card has to be named `card.md` for the build process to work. Same change needed for each card.

Also, as a convention, images should be named `image-1.png`, `image-2.png`, etc.